suresh/recursion-practice.js

```diff
@@ -0,0 +1,92 @@
+/* 1) Create a function to get the factoral of a number.
+  Input: factoral(5)
+  Output: 120 */
+  
+function factoral(no) {
+  var j = 1;
+  
+  for (var i = 1; i <= no; i++) {
+    j = j * i;
+  }
+  return(j);
+}
+ 
+console.log(factoral(4));
+ 
+function factoral1(no) {
+  if (no < 0) {
+   return 'invalid number';
+  } else if (no === 0 || no === 1) {
+    return 1;
+    } else {
+      return no * factoral1(no-1);
+      }
+}
+
+console.log(factoral1(6));
+ 
+/* 2) Create a function to get the power of a number. Multiply the first input x(second input) number of times 
+   Input: power(2, 5)
+   Output: 32  
+   reason: 2x2x2x2x2*/
+ 
+function power(x, y) {
+  if (x === 0 || y === 0) {
+    return 0;
+    } else if (x === 1 && y >1) {
```

Comment:
your else if statements should be on the same indent as the initial if

Reply:
power is taking the first number and multipying it by itself `y` many times. Double check your conditions